spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -951,29 +951,34 @@ def a; end
       YAML
     end
 
-    context 'when configurable parameter is an obsolete parameter' do
-      it 'displayed as a new parameter setting value without duplication' do
+    context 'when duplicated default configuration parameter' do
+      before do
+        RuboCop::ConfigLoader.default_configuration['Naming/MethodParameterName']
+                             .merge!('AllowedNames' => %w[at by at])
+      end
```

Comment:
The configuration values like `# AllowedNames: at, by` that appear in `.rubocop_todo.yml` were not configuration values in `.rubocop.yml`, but default values in `config/defautl.yml`.
However, what appeared to be the `.rubocop.yml` configuration values in #10875 . This was due to the default values being corrupted by the problem fixed in this PR.
Therefore, it was necessary to duplicate parameter the `config/defautl.yml` here.